src/EncompassRest/CustomDataObjects/CustomDataObjects.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using EncompassRest.Utilities;
+
+namespace EncompassRest.CustomDataObjects
+{
+    public class CustomDataObjects : ApiObject
```

Comment:
why is this not abstract?

Reply:
It's used directly for Global

Reply:
That seems reasonable to me. I'll add a global version as well.